src/OpenLoco/src/Map/SurfaceElement.h

```diff
@@ -63,7 +63,7 @@ namespace OpenLoco::Map
             setType(kElementType);
             setBaseZ(baseZ);
             setClearZ(clearZ);
-            _flags = quarterTile;
+            _flags = static_cast<ElementFlags>(quarterTile);
```

Comment:
@ZehMatt is this ok here? Is there a way to use these enum flags for `_flags`? seems hard because its in a base class used by multiple other enums

Reply:
The problem is that _flags stores more than just the ElementFlags, as I stated in a prior comment it would be best to have getter/setter to specifically set the flags. I'm not sure what the other bits are used for but this should just become another getter/setter so we don't have to access _flags directly and once interop is gone we can split the variable up.

Reply:
This also means we can adjust the ElementFlags values accordingly so they start with the first bit since how its stored shouldn't matter for the flags.

Reply:
> This also means we can adjust the ElementFlags values accordingly so they start with the first bit since how its stored shouldn't matter for the flags.

Do you want me to make this change?

Reply:
That would be good. It should use addFlags here where you first extract the actual flags out of the quarterTile argument.

Reply:
> That would be good. It should use addFlags here where you first extract the actual flags out of the quarterTile argument.

I think I did what you asked, but please take a look to verify.